include/translated-object.php

```diff
@@ -398,14 +414,48 @@ public function where_clause( $lang ) {
 	}
 
 	/**
-	 * Check if a user can synchronize translations.
+	 * Returns ids of objects in a language similarly to get_objects_in_term() for a taxonomy.
+	 * It is faster than get_objects_in_term() as it avoids a JOIN.
+	 *
+	 * @since 1.4
+	 *
+	 * @param PLL_Language $lang PLL_Language object.
+	 * @return int[] Object ids.
+	 */
+	public function get_objects_in_language( $lang ) {
+		global $wpdb;
+		$tt_id = $this->tax_tt;
+
+		$last_changed = wp_cache_get_last_changed( 'terms' );
+		$cache_key    = "polylang:get_objects_in_language:{$lang->$tt_id}:{$last_changed}";
+		$cache        = wp_cache_get( $cache_key, 'terms' );
+
+		if ( false !== $cache ) {
+			return $this->sanitize_int_ids_list( $cache );
+		}
+
+		$object_ids = $wpdb->get_col( $wpdb->prepare( "SELECT object_id FROM $wpdb->term_relationships WHERE term_taxonomy_id = %d", $lang->$tt_id ) );
+		$object_ids = $this->sanitize_int_ids_list( $object_ids );
+		wp_cache_set( $cache_key, $object_ids, 'terms' );
+
+		return $object_ids;
+	}
```

Comment:
This method was removed in #980. It's coming back probably because a merge conflict. I propose to remove it again.

Reply:
Done in [19b73bf](https://github.com/polylang/polylang/pull/984/commits/19b73bfc175a4559d04798eb348343a61feadcea).